edalize/symbiyosys.py

```diff
@@ -108,8 +112,10 @@ class Symbiyosys(Edatool):
         "lists": {
             # A list of tasks to run from the .sby file. Passed on the sby
             # command line.
-            "tasknames": "String"
-        }
+            "tasknames": "String",
```

Comment:
Is this `tool_options` some legacy stuff? Other edatool classes are using `TOOL_OPTIONS` with a different structure.

Reply:
This backend is part of the legacy tool API. The classes under the `tools` and `flows` directories belong to the new flow API. Unfortunately most tools haven't been ported over to the new API yet.